addons/account/views/account_move_views.xml

```diff
@@ -962,29 +962,51 @@
                                 <label for="journal_id"
                                        groups="account.group_account_readonly"
                                        invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'"/>
+                                <label for="currency_id"
+                                       groups="!account.group_account_readonly,base.group_multi_currency"
+                                       invisible="move_type == 'entry'"/>
                                 <div name="journal_div"
                                      class="d-flex"
-                                     groups="account.group_account_readonly"
                                      invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'">
                                     <field name="journal_id"
                                         options="{'no_create': True, 'no_open': True}"
+                                        groups="account.group_account_readonly"
                                         readonly="posted_before"/>
-                                    <span class="oe_inline o_form_label mx-3"
-                                          groups="base.group_multi_currency"
-                                          invisible="move_type == 'entry'"> in </span>
-                                    <field name="currency_id"
-                                           groups="base.group_multi_currency"
-                                           invisible="move_type == 'entry'"
-                                           readonly="state != 'draft'"
-                                           options="{'no_create': True}"
-                                           context="{'search_default_active': 1, 'search_default_inactive': 1}"/>
+                                    <div groups="account.group_account_readonly">
+                                        <span class="oe_inline o_form_label mx-3"
+                                              groups="base.group_multi_currency"
+                                              invisible="move_type == 'entry'"> in </span>
+                                    </div>
+                                    <div name="currency_div"
+                                         groups="base.group_multi_currency"
+                                         invisible="move_type == 'entry'">
+                                        <field name="currency_id"
+                                               nolabel="1"
+                                               readonly="state != 'draft'"
+                                               options="{'no_create': True}"
+                                               context="{'search_default_active': 1, 'search_default_inactive': 1}"/>
```

Comment:
Is there no way to avoid repeating the field in the below div?

Reply:
I don't think so. All the "dynamic" owl directives (t-out, t-set, ...) are forbidden [^1] and we do need to display the information twice.
But if you have any suggestions I would be curious.
[^1]: https://github.com/odoo/odoo/blob/7369cc291f5f30b8b312d6c1f9dada3fe83e06b3/odoo/addons/base/models/ir_ui_view.py#L1807

Reply:
Maybe something like this? (to validate with PO)
When no currency is set (or when it is the company currency)
```markdown
Journal     [Invoices   ] in [USD ]
```

When a currency is set
```markdown
Journal     [Invoices   ] with (1 [EUR ] = 2 USD)
```

Reply:
I checked with labe and we would rather keep the selection and the conversion information separate to make it more understandable. It seems to be the common practice.